tlatools/org.lamport.tlatools/customBuild.xml

```diff
@@ -139,15 +139,6 @@
 
 	<!-- Compiles the TLA+ tools code -->
 	<target name="compile" depends="clean, generate">
-		<echo>
-			================================================================
-			= The following warnings about sun.misc.Unsafe can be ignored. = 
-			= OffHeapDiskFPSet has been written using Unsafe to best use   =
-			= a computer's memory. If Unsafe is removed from the JVM,      =
-			= OffHeapDiskFPSet won't work anymore.                         =
-			================================================================
-		</echo>
-		<!-- compile -->
```

Comment:
The message of this warning remains valid even if warnings are disabled. It should not be removed but reworded to inform users if compilation fails due to an upcoming JVM update that removes Unsafe.